### PR TITLE
 Karma response emojis now configurable

### PR DESCRIPTION
**Pull request by jcorrado**
@devth I'll rebase to master after you merge #88

### CODE REVIEW

**Comment by jcorrado** [comment]
This is the only commit in this, after future rebase:
https://github.com/yetibot/yetibot.core/pull/89/commits/690a881bfa81c3f60fd2a617c1c4246bfec4b998

File: config/config.sample.edn [hunk 1]
```diff
@@ -66,4 +66,5 @@
   :imgflip {:username "", :password ""},
   :admin {:commands ["observer" "obs"]
           :users ["U123123" "~awesomeperson"]},
-  :nrepl {:port ""}}}
+  :nrepl {:port ""}
+  :karma {:emoji {:positive "🌮" :negative "💩"}}}}
```
Comment (devth) [inline config/config.sample.edn]: I know it's a pain but can you add the samples to the alternative config format in `config/profiles.sample.clj`? In the future we'll autogen these after we migrate to spec.

File: src/yetibot/core/commands/karma.clj [hunk 2]
```diff
@@ -1,60 +1,61 @@
...
    [yetibot.core.commands.karma.specs :as karma.spec]
    [clojure.string :as str]
-   [clj-time.format :as fmt]
+   [clj-time.format :as t]
    [clojure.spec.alpha :as s]))
 
+(def config (:value (get-config sch/Any [:karma])))
+
+(def pos-emoji (or (-> config :emoji :positive) "🌈"))
+(def neg-emoji (or (-> config :emoji :negative) "⛈"))
```
Comment (devth) [inline src/yetibot/core/commands/karma.clj]: I wonder if/how we could expose the react-based method to the user in docs. Maybe a good candidate for more general docs once we revamp `help`. 🤔

File: src/yetibot/core/commands/karma/specs.clj [hunk 3]
```diff
@@ -1,33 +1,32 @@
 (ns yetibot.core.commands.karma.specs
   (:require [clojure.spec.alpha :as s]))
 
-;; Common specs we could relocate
 (s/def ::name string?)
 (s/def ::id string?)
 (s/def ::user (s/keys :req-un [::name ::id]))
 
-(s/def ::adapter #{:graphql :irc :slack})
```
Comment (devth) [inline src/yetibot/core/commands/karma/specs.clj]: What's the reasoning on removing chat-source and adapter specs? Just curious.

Comment (jcorrado) [reply src/yetibot/core/commands/karma/specs.clj]: I was using adapter details to drive formatting decisions.  After removing that formatting logic, the adapter key value was no longer used anywhere, so I cleaned it out of the spec.  I figured it's better to prune it than to leave in place, unused. Easy enough to add back in if we want to generalize the specs a bit and pull most of them out of Karma.

**Pull request merged**